CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.6.0] - 2025-03-12
```

Comment:
you need to set the pom.xml's to the new version and push: 
```
mvn versions:set -DgenerateBackupPoms=false -DnewVersion=3.6.0-SNAPSHOT    
```
Then run the release action and it will tag to 3.6.0 and increase to 3.6.1-SNAPSHOT.